fe/fe-core/src/main/java/com/starrocks/catalog/PrimitiveType.java

```diff
@@ -273,6 +297,142 @@ public static int getDefaultScaleOfDecimal(PrimitiveType t) {
         }
     }
 
+    // TODO(mofei) refactor it
+    public static String getUdfTypeName(PrimitiveType t) {
```

Comment:
Could remove this method directly.

Reply:
It's used by `com.starrocks.analysis.InPredicate#initBuiltins`. It was migrated to here to keep compact to type definition.